load-issues.js

```diff
@@ -0,0 +1,21 @@
+var Issue = require('./issue.js');
+var config = require('./config.js');
+var debug = require('debug')('backlog:load-issues');
+var github = require('./github.js');
+
+module.exports = function getIssues() {
```

Comment:
FYI, `x = function f() { … }` doesn't define the name `f`, even in an uninitialized state; it just defines `x` and assigns an anonymous function to it.

```js
module.exports = function f(x) {
  console.log(x)
}

console.log(f) // Error
```

Reply:
Kinda, it actually defines it within the function itself so you can do recursion. It also shows up in stack traces instead of "anonymous function".  In this case, I did it for the sake of the human reader.

https://stackoverflow.com/a/3883869/13216